tests/NotNullViolationError.js

```diff
@@ -41,6 +41,10 @@ module.exports = (session) => {
             if (session.isPostgres() || session.isSqlite()) {
               expect(error.table).to.equal(table);
             }
+
+            if (session.isMssql()) {
+              expect(error.table).to.equal('master.dbo.' + table);
```

Comment:
Also here, why is there the `master.dbo` prefix?

Reply:
So yeah, it's `database.schema`. If you feel that information is needed, I think we should add a separate properties for `database` and `schema`. That way `table` is the same across all databases (when it exists)